src/lib/core/components/GlobalFiltersDialog.tsx

```diff
@@ -22,6 +23,7 @@ interface Props {
   setFilters: (filters: Filter[]) => void;
   // A function to remove a given filter
   removeFilter: (filter: Filter) => void;
+  variableLinkConfig: VariableLinkConfig;
```

Comment:
**nitpick**: Might want to add a comment for uniformity. Maybe something like: "a configuration option allowing items in the VariableList to be either buttons or anchors."

Reply:
Ah, yes. I'll add an appropriate comment